src/events/UncheckZone.java

```diff
@@ -0,0 +1,16 @@
+package events;
+
+public class UncheckZone extends SecurityEvent {
+	private static UncheckZone instance;
+
+	private UncheckZone() {
+
+	}
+
+	public static UncheckZone instance() {
+		if (instance == null) {
+			instance = new UncheckZone();
+		}
+		return instance;
+	}
+}
```

Comment:
Need a new line after the last bracket

Reply:
Oh, I'm all for compacting the code, only I can't address it that quickly now, because I started coding further, so in the next PR...